integration/test.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/bash
+
+NUMBER=$RANDOM
+curl -i -X GET -H 'Content-Type: application/json' localhost:3000/ > GET
```

Comment:
We should bail at the first error:

```suggestion
#!/usr/bin/bash

set -x

NUMBER=$RANDOM
curl -i -X GET -H 'Content-Type: application/json' localhost:3000/ > GET
```